packages/prescriptionSearch/src/schema/request.ts

```diff
@@ -0,0 +1,62 @@
+import {FromSchema, JSONSchema} from "json-schema-to-ts"
+
+export const bundleEntrySchema = {
+  "type": "object",
+  "required": [
+    "fullUrl",
+    "resource",
+    "request"
+  ],
+  "description": "A FHIR collection Bundle.",
+  "properties": {
+    "fullUrl": {
+      "type": "string",
+      "examples": ["urn:uuid:4d70678c-81e4-4ff4-8c67-17596fd0aa46"]
+    }
+  }
+} as const satisfies JSONSchema
+
+export const bundleSchema = {
```

Comment:
There won't be a request bundle so we can just delete this file.

Reply:
The unnecessary schema has been removed.